src/x86_const.rs

```diff
@@ -244,6 +244,15 @@ pub enum RegisterX86 {
     R13W,
     R14W,
     R15W,
+    IDTR,
+    GDTR,
+    LDTR,
+    TR,
+    FPCW,
+    FPTAG,
+    MSR,
+    MXCSR,
+    ENDING
```

Comment:
ENDING is an end marker, we don't need it. Please remove it .

Reply:
removed in latest commit